src/components/content-tab-overview.tsx

```diff
@@ -181,6 +187,36 @@ function ShortcutsSection( { selectedSite }: Pick< ContentTabOverviewProps, 'sel
 	return <ButtonsSection buttonsArray={ buttonsArray } title={ __( 'Open in…' ) } />;
 }
 
+function PublishBanner( {
+	selectedSite: _selectedSite,
+}: Pick< ContentTabOverviewProps, 'selectedSite' > ) {
+	const { __ } = useI18n();
+	const dispatch = useAppDispatch();
+	const { setSelectedTab } = useContentTabs();
+	const { connectedSites } = useConnectedSitesData();
+
+	const handlePublishClick = () => {
+		// Navigate to Sync tab and open modal with 'push' mode
+		setSelectedTab( 'sync' );
+		dispatch( connectedSitesActions.openModal( 'push' ) );
+	};
+
+	// Only show banner if site has no connected sites
+	if ( connectedSites.length > 0 ) {
+		return null;
+	}
+
+	return (
+		<div className="w-full flex items-center gap-3 px-4 py-3 rounded-sm bg-a8c-gray-0 border border-a8c-gray-5">
+			<Icon icon={ info } size={ 20 } className="text-a8c-blue-50 flex-shrink-0" />
+			<p className="flex-1 text-black">{ __( 'Your site is ready for publishing' ) }</p>
```

Comment:
Let's rename the message into a question:

```suggestion
			<p className="flex-1 text-black">{ __( 'Ready to go live?' ) }</p>
```

Reply:
Ok, done in e2a165a